src/Kros.MediatR.Extensions/README.md

```diff
@@ -0,0 +1,18 @@
+# Kros.MediatR.Extensions
+
+__Kros.MediatR.Extensions__ je knižnica, ktorá rozširuje knižnicu [MediatR]([https://link](https://github.com/jbogard/MediatR)).
+
+### NullCheckPostProcessor
+Registrovaním `service.AddMediatRNullCheckPostProcessor()` pridáte do MediatR pipeline-y post processor, ktorý kontroluje výsledok requestu a ak je `null` tak vyvolá `NotFoundException`.
+
+### ControllerBaseExtensions
+Rozširuje `ControllerBase` o veci, ktoré zjednodušujú prácu s MediatR v controllery.
+>Napríklad: zavolať požiadavku cez MediatR môžte nasledovne:
```

Comment:
It is not required (no warning in markdown linter), but I would surround quote block by empty lines.